app/tests/integration/nimbus/test_e2e_create_experiment.py

```diff
@@ -193,12 +193,12 @@ def test_create_new_experiment_remote_settings_reject(selenium, base_url):
     modal.decline_changes()
 
     # Load home page and wait for experiment to show in the Drafts tab
```

Comment:
This gets the failing integration test to pass, but it's kind of a weird fix. 

The thing that it was tripping on was `HomePage(selenium, base_url).wait_for_page_to_load()` - and as far as I can tell, that just looks for the presence of `.directory-table` in the DOM.

As far as I can tell, that element is in the DOM when it fails. And this same method is used in other tests and works. But, for some reason, it consistently fails in this test.